### DISCUSSION TITLE
RadioLib compile errors

### DISCUSSION THREAD

**Discussion by veronal** [timmbogner/Farm-Data-Relay-System#199] [General]
A while back my Radiolib library updated (v6.4.2).
Yesterday I tried recompiling a previously working sketch which is now throwing errors
[CompileError.txt](https://github.com/timmbogner/Farm-Data-Relay-System/files/14658955/CompileError.txt)

Anyone got any ideas as to what the problem could be ?
Thanks in advance ..

**Comment by aviateur17** [comment]
@veronal, not sure exactly what the issue is.  I see that those lines of code that are shown in the compile error were modified in the 6.4.2 update of RadioLib but I'm guessing he has done compilation testing and upon searching for similar issues I didn't see anything pop up.   I wonder if you have any special RadioLib #defines in your project?  like RADIOLIB_GODMODE or anything like that defined?   Otherwise, perhaps try removing or deleting all of the RadioLib libraries that you have installed in Arduino and reinstall them.

**Reply by timmbogner** [reply]
Hey @veronal , I can't reproduce the issue either. I agree that you should delete and re-install RadioLib. It would also be good to see your sketch files, specifically your RadioLib configuration settings.

**Reply by veronal** [reply]
Thanks, I tried it with one one of the default FDRS Lora sketches and still got the error, so it seems to be systemic.
I will try re-installing the library.. thanks for your help.

**Reply by veronal** [reply]
As an FYI I was able to successfully compile after re-installing the RadioLib library.
Thanks again for the assist
